project/ws/app/src/lib/routes/home/pipes/public-gcp-url.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core'
+import { environment } from '../../../../../../../../src/environments/environment'
+// import { environment } from '../../../../../../../../src/environments/environment'
+
+@Pipe({
+  name: 'publicGcpUrl',
+})
+export class PublicGcpUrlPipe implements PipeTransform {
+  transform(url: string): string {
+    const originalUrlPart = 'https://storage.googleapis.com/igotuat'
```

Comment:
please confirn, it will be changing as per the environment

Reply:
Yes Vishnu, environment URL is dynamic.